universum/modules/code_report_collector.py

```diff
@@ -95,23 +95,23 @@ def report_code_report_results(self) -> None:
             json_file: TextIO = self.artifacts.create_text_file("Static_analysis_report.json")
             json_file.write(json.dumps(report, indent=4))
 
-            issue_count: int = 0
+            issue_count: int
             if report or report == []:
                 try:
                     issue_count = self._report_as_sarif_json(report)
                 except (KeyError, AttributeError, ValueError):
                     try:
                         issue_count = self._report_as_pylint_json(report)
                     except (KeyError, AttributeError, ValueError):
-                        self.out.log_stderr("Could not parse report file. Something went wrong.")
+                        self.out.log_error("Could not parse report file. Something went wrong.")
                         continue
             else:
-                self.out.log_stderr("There are no results in code report file. Something went wrong.")
+                self.out.log_error("There are no results in code report file. Something went wrong.")
                 continue
 
             if issue_count != 0:
                 text = str(issue_count) + " issues"
-                self.out.log_stderr("Found " + text)
-                self.out.report_build_status(os.path.splitext(os.path.basename(report_file))[0] + ": " + text)
+                self.out.log_error("Found " + text)
+                self.out.set_build_title(os.path.splitext(os.path.basename(report_file))[0] + ": " + text)
```

Comment:
"Build title" can be mistaken for build tag. I guess this thing might be called 'status line' or something like that. Maybe, 'inline report'?

Reply:
It was previously called "build status" and I deliberately changed it.
This is because it is literally a title of this build on the build history page in the TeamCity.

Reply:
TeamCity says just 'Status', so we can use the official naming.